src/typing/env_js.ml

```diff
@@ -44,7 +48,10 @@ let swap_changeset f =
   changeset := newset;
   oldset
 
-let global_scope: scope = ref SMap.empty
```

Comment:
Err, I already merged that locally. Thought I pushed, but apparently I didn't. I'll push that up now.